CMakeLists.txt

```
@@ -224,7 +224,9 @@ add_subdirectory (gridcomps)
 add_subdirectory (griddedio)
 if (BUILD_WITH_FARGPARSE)
    add_subdirectory (tutorial)
-endif()
+ endif()
+ add_subdirectory (benchmarks)
+ 
```

```suggestion
   add_subdirectory (benchmarks)
endif()
```

It looks like the benchmarks use fArgParse, so we want to make sure they are only built if we build with fArgParse.